packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -883,23 +885,29 @@ export class BasePicker<T, P extends IBasePickerProps<T>>
     return currentIndex > -1 && !this.state.suggestionsLoading ? 'sug-' + currentIndex : undefined;
   }
 
-  protected getSuggestionsAlert(suggestionAlertClassName: string = legacyStyles.screenReaderOnly) {
-    const currentIndex = this.suggestionStore.currentIndex;
-    if (this.props.enableSelectedSuggestionAlert) {
+  protected renderCustomAlert(alertClassName: string = legacyStyles.screenReaderOnly) {
+    const { enableSelectedSuggestionAlert, suggestionRemovedText = 'removed' } = this.props;
+    let selectedSuggestionAlertText = '';
+    let removedItemText = '';
+
+    if (enableSelectedSuggestionAlert) {
+      const currentIndex = this.suggestionStore.currentIndex;
       const selectedSuggestion =
         currentIndex > -1 ? this.suggestionStore.getSuggestionAtIndex(this.suggestionStore.currentIndex) : undefined;
-      const selectedSuggestionAlertText = selectedSuggestion ? selectedSuggestion.ariaLabel : undefined;
-      return (
-        <div
-          className={suggestionAlertClassName}
-          role="alert"
-          id={this._ariaMap.selectedSuggestionAlert}
-          aria-live="assertive"
-        >
-          {selectedSuggestionAlertText}{' '}
-        </div>
-      );
+      selectedSuggestionAlertText =
+        selectedSuggestion && selectedSuggestion.ariaLabel ? selectedSuggestion.ariaLabel : '';
     }
+
+    if (this.state.selectionRemoved) {
+      removedItemText = `${suggestionRemovedText} ${this._getTextFromItem(this.state.selectionRemoved, '')}`;
```

Comment:
I wonder if this should be a string with a placeholder (`removed {0}`) in case of different order in other languages?

Reply:
I was debating it, but went the easier route 😅. I can definitely do that though!